sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -119,38 +117,24 @@ int _handle_reply(gnrc_pktsnip_t *pkt, uint64_t time)
     return 1;
 }
 
-static inline void _a_to_timex(timex_t *delay, const char *a)
-{
-    int ms = atoi(a);
-
-    if (ms >= 0) {
-        delay->seconds = 0;
-        delay->microseconds = ms * 1000;
-        timex_normalize(delay);
-    }
-}
-
-static void _print_stats(char *addr_str, int success, int count, timex_t stop,
-                        timex_t sum_rtt, timex_t min_rtt, timex_t max_rtt)
+static void _print_stats(char *addr_str, int success, int count, uint64_t total_time,
+                         uint64_t sum_rtt, uint64_t min_rtt, uint64_t max_rtt)
 {
     printf("--- %s ping statistics ---\n", addr_str);
 
     if (success > 0) {
-        timex_normalize(&sum_rtt);
+        uint32_t avg_rtt = (uint32_t)sum_rtt / count;  /* get average */
         printf("%d packets transmitted, %d received, %d%% packet loss, time %"
-                PRIu32 ".06%" PRIu32 " s\n", count, success,
-                (100 - ((success * 100) / count)), stop.seconds, stop.microseconds);
-        timex_t avg_rtt = timex_from_uint64(timex_uint64(sum_rtt) / count);  /* get average */
+               PRIu32 ".06%" PRIu32 " s\n", count, success,
+               (100 - ((success * 100) / count)),
+               (uint32_t)total_time / SEC_IN_USEC, (uint32_t)total_time % SEC_IN_USEC);
```

Comment:
casting the `total_time` here to `uint32_t` may not show the proper `total_time` if a long-term ping exceeds > ~70 min